### ISSUE TITLE
Make footer floating

### ISSUE THREAD

**Issue by vprusso** [unitaryfund/metriq-app#312]
Right now, the footer is only viewable when scrolling all the way to the bottom of the screen. If we have a substantial number of submissions, this may be difficult for users to see. Make the footer floating so that it is viewable at any point in the infinite scroll.

**Comment by WrathfulSpatula** [comment]
Kass' redesign of the footer would help somewhat, here, in that it isn't as tall. I'm starting there, then. This is a popular request, among the Unitary Fund team, but, without simultaneous consideration to the header and the size of the viewport left over, it's not going to look how we think. I'll figure it out, but neither or both header and footer should float, and the viewport should still be reasonable, and it has to stay responsive for mobile.

**Comment by WrathfulSpatula** [comment]
We're making progress, here, with a much smaller footer, but I'm diverting to the enhancement request we received, to allow users to edit results rows. This issue is open, for the moment.

**Comment by WrathfulSpatula** [comment]
This seems to a high-priority request, as an issue, but the only reason I haven't implemented it yet if that we simply don't have enough screen space for our large footer, particularly if we decide it doesn't look right without making the header section sticky, and _definitely_ not on mobile.

I anticipate that this is going to entail using React.js and CSS media queries to partially collapse the (sticky) footer unless the user mouse cursor hovers over it, or until the user scrolls all the way to the bottom. We might have to completely disable it on mobile, or at least collapse it even further. Also, we can't actually reliably detect whether the user is on mobile, just either viewport size in JavaScript (which is probably an anti-pattern, to rely on directly to update the DOM) or CSS media queries (which are probably the "right" way to do this, but they don't directly control DOM layout).

All of this makes for a really complicated view component. None of the above would be necessary if the footer (and header) were significantly less tall, but there's not much else we can cut, (and we've rather than been adding to the height of the footer, since last approaching this issue).

TL:DR; It might be a surprise, but this is comparatively "nontrivial," and it might not look the way we expect. However, maybe I can complete something the acceptably fulfills the issue intent, today.

**Comment by WrathfulSpatula** [comment]
I did my best to achieve this issue's intent, with PR #481. When @vprusso approves the code, we'll publish to QA, and the team can took a look and critique there. As I suspected, the footer is _way_ too to simply set a fixed position at bottom of screen, without some kind of animated collapsible transitions to stick to bottom without resting in the way of the viewport.
